content/hardware/05.nicla/boards/nicla-sense-me/tutorials/nicla-battery-app-note/nicla-battery-app-note.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Understanding Battery Capabilities Of The Nicla Sense ME'
+difficulty: intermediate
+description: 'Make use of the BQ2 your Nicla application.'
+tags:
+  - Battery
+  - Nicla
+  - BQ25120
+
+author: 'Ali Jahangiri'
+hardware:
+  - hardware/01.mkr/05.boards/nicla-sense-me
+software:
+  - Arduino IDE 2.x
+---
+
+## Introduction
+
+We have developed the Nicla Sense ME for low-power operation, and it really shines when you can use it independantly of a USB cable, especially for wireless sensing applications. The Nicla Sense ME includes a low-profile connector that will allow you to connect a LiPo battery. In this application note, we will take a closer look at the battery capabilities of the Nicla boards and how you can use it effectively.
+
+### Goals
+The goals of this project are to:
+- Learn how to identify connector polarity
+- Understand the features of the BQ25120 IC
+- Set charging current on the Nicla Sense ME
+- 
+
+### Hardware & Software Needed
+- [Arduino Nicla Sense ME](https://store.arduino.cc/products/nicla-sense-me)
+- Multimeter
+- IDE 2.x
+- LiPo battery with 3-pin 1.2mm ACH Battery Male Connector
+
+### NTC Battery protection
+You can learn about Li-Ion batteries in the [MKR Battery Application Note](/tutorials/mkr-wifi-1010/mkr-battery-app-note). Briefly, the Nicla Sense ME uses a single cell Li-Ion battery. However, this must include a NTC cable, and also comes with a different connector to save space.
+
+![Nicla Battery Breakdown](assets/3-pin-lipo-battery-breakdown.png)
+
+The + wire connects to the battery, while the - terminal connects to the ground. The NTC pin connects the negative thermal constant (NTC) resistor to the Nicla board. The NTC helps the Nicla board identify the temperature of the battery and disables it if it is outside the operating temperature. The resistance of the resistor, as the name suggests, becomes less at higher temperatures. Commonly, the resistance between the NTC pin and ground is 10k.
+
+*** Try measuring the resistance between the NTC pin and the ground with a multimeter. Make sure not to short the pins of the battery. ***
+
+### Connector
+As noted, a special lower footprint battery connector is used to connect the battery to the Nicla Sense ME board. A male 3-pin JST-ACH connector with 1.2mm pitch must be avaliable on the battery. Each individual connector is made of one plastic housing and three metal crimp terminals. A crimping device may be required if you wish to prepare your own wires. Note that when looking from above (with the slots facing downwards), the red (positive) wire should be on your left.
```

Comment:
Since this is not a common connector used in batteries, I would add a buying reference for the audience; I would also add some references to this connector, such as its datasheet and the manufacturer's website with te connector's dimensional and technical information.

Reply:
It was decided that this is not going to be included.